src/states/node.rs

```diff
@@ -516,8 +516,8 @@ impl Node {
                 if let Ok(public_id) = verify_signed_public_id(serialised_public_id, signature) {
                     self.handle_client_identify(public_id, peer_id, client_restriction).to_evented()
                 } else {
-                    warn!("{:?} Signature check failed in ClientIdentify - Dropping connection \
-                           {:?}",
+                    warn!("{:?} Signature check failed in ClientIdentify, so dropping connection \
+                           {:?}.",
```

Comment:
I actually like the version with a hyphen more, but that may be just my personal preference ;) The same goes for other similar changes.